### PR TITLE
Js without assigned to jd 317

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#317
Pagination in Agency people homepage : JobSeeker's without Jobdeveloper

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
PR for review

File: app/controllers/agency_people_controller.rb [hunk 1]
```diff
@@ -190,6 +192,29 @@ def list_js_jd
...
+  def list_js_without_jd
+
+    raise 'Unsupported request' if not request.xhr?
+       
+    @agency_person= AgencyPerson.find(params[:id])
+
+    @people_type_without_jd = params[:people_type] || 'jobseeker-without-jd'
+
+    @people = []
+    @people = display_job_seekers @people_type_without_jd, @agency_person
```
Comment (patmbolger) [inline app/controllers/agency_people_controller.rb]: The variables @agency_person, @people_type_without_jd, @people are passed directly into the partials via the 'locals' hash, so these vars don't need to be instance vars and could be simply local vars for this method.  I suspect that as a result there is a little more overhead occurring than we need (Rails detects controller instance vars and makes those vars available to the view - even though the controller and view are *not* related via class structure - this requires work that can be avoided with local vars).

**Comment by patmbolger** [comment]
Reviewed PR .... looks good.  

I do have one suggested cleanup related to controller instance vars.

File: app/controllers/agency_people_controller.rb (hunk 1 above)
Comment (joaopapereira) [inline app/controllers/agency_people_controller.rb]: Good point here

File: config/routes.rb [hunk 2]
```diff
@@ -44,6 +44,13 @@
              'agency_people#list_js_jd', as: :list_js_jd_agency_people
 
 
+
+  get 'agency_people/:id/list_js_without_jd/:people_type' => 
+              'agency_people#list_js_without_jd',as: :list_js_without_jd_agency_people
+
```
Comment (joaopapereira) [inline config/routes.rb]: I think we can remove these extra spaces as well

**Comment by joaopapereira** [comment]
Just finished my review, and the PR looks good, please just revisit my comments and Patrick's and I believe we are done

File: config/routes.rb (hunk 2 above)
Comment (vijayakarumudi) [inline config/routes.rb]: I removed the space.

File: app/controllers/agency_people_controller.rb (hunk 1 above)
Comment (vijayakarumudi) [inline app/controllers/agency_people_controller.rb]: I updated variables.

**Comment by vijayakarumudi** [comment]
updated PR.

**Comment by patmbolger** [comment]
Reviewed PR again .... looks good.

**Comment by joaopapereira** [comment]
Looks good, going to merge

**Pull request merged**